daemons/fenced/fenced_remote.c

```diff
@@ -2226,7 +2271,7 @@ fenced_process_fencing_reply(xmlNode *msg)
     crm_trace("Next for %s on behalf of %s@%s (result was: %s)",
               op->target, op->originator, op->client_name,
               pcmk_exec_status_str(result.execution_status));
-    call_remote_stonith(op, NULL, pcmk_rc2legacy(stonith__result2rc(&result)));
+    request_peer_fencing(op, NULL, &result);
```

Comment:
do we need to reset the result here?

Reply:
whoops, yes

Reply:
added commit on top to fix memory issues